### PR TITLE
Feat(line-chart): dropdown range selector

### PR DESCRIPTION
**Pull request by tlebon**
- [x] added dropdown select menu for mobile line chart range

### CODE REVIEW

File: src/pages/AuctionPage/AuctionPage.tsx [hunk 1]
```diff
@@ -64,18 +64,21 @@ export const AuctionPageComponent: React.FC<AuctionPageProps> = ({ market }) =>
...
-      color: theme.palette.success.main,
-      data: [],
-    },
-    {
-      id: 'No',
-      color: theme.palette.error.main,
-      data: [],
-    },
-  ];
+  const initialData: Serie[] = React.useMemo(
```
Comment (shekhar-shubhendu) [inline src/pages/AuctionPage/AuctionPage.tsx]: remove useMemo as this breaks the chart

File: src/pages/MarketPage/MarketPage.tsx [hunk 2]
```diff
@@ -177,18 +177,21 @@ export const MarketPageComponent: React.FC<MarketPageProps> = ({ market }) => {
...
-      color: theme.palette.success.main,
-      data: [],
-    },
-    {
-      id: 'No',
-      color: theme.palette.error.main,
-      data: [],
-    },
-  ];
+  const initialData: Serie[] = React.useMemo(
```
Comment (shekhar-shubhendu) [inline src/pages/MarketPage/MarketPage.tsx]: remove useMemo as this breaks the chart

**Comment by shekhar-shubhendu** [comment]
Also, I think it should be a native html5 select not material-ui select so that it triggers native mobile selector.
Please check this with @anitasengupta

**Comment by tlebon** [comment]
ill see if @anitasengupta can leave a quick comment on Monday (maybe around our FE meeting).

**Review by shekhar-shubhendu — DISMISSED** [review]

**Comment by tlebon** [comment]
this should be done now

**Review by shekhar-shubhendu — APPROVED** [review]

**Pull request merged**